recipes/IEMOCAP/emotion_recognition/train_with_wav2vev2.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python3
+"""Recipe for training an emotion recognition system from speech data only using IEMOCAP.
+The system classifies 4 emotions ( anger, happiness, sadness, neutrality) with wav2vec2.
+
+To run this recipe, do the following:
+> python train_with_wav2vec2.py hparams/train_with_wav2vec2.yaml --data_folder /path/to/IEMOCAP
+
+Authors
+ * Yingzhi WANG 2021
+"""
+
+import os
+import sys
+import speechbrain as sb
+import torch
+from hyperpyyaml import load_hyperpyyaml
+
+
+class EmoIdBrain(sb.Brain):
+    def compute_forward(self, batch, stage):
+        """Computation pipeline based on a encoder + emotion classifier.
+        """
+        batch = batch.to(self.device)
+        wavs, lens = batch.sig
+
+        wav2vec2_out = self.modules.wav2vec2(wavs)
+
+        average_pooling = sb.nnet.pooling.Pooling1d(
```

Comment:
Yingzhi, Could you please use:
AdaptativeAvgPool1D instead of sb.nnet.pooling.Pooling1d

you can do it in that way:
create you obj in the yaml:
```
avg_pool: !new:torch.nn.AdaptiveAvgPool1d
              output_size:1
```
and in you python file
```
# last dim will be used for AdaptativeAVG pool
input = input.permute(0,2,1)
out = this.avg_pool(input)
out.view(out.shape[0],-1)
```

Reply:
Done